cloudaux/orchestration/aws/s3.py

```diff
@@ -313,31 +365,21 @@ def get_bucket(bucket_name, output='camelized', include_created=False, **conn):
 
     conn['region'] = region
 
-    grants, grant_refs, owner = get_grants(bucket_name, include_owner=True, **conn)
-
     result = {
         'arn': "arn:aws:s3:::{name}".format(name=bucket_name),
-        'grants': grants,
-        'grant_references': grant_refs,
-        'owner': owner,
-        'lifecycle_rules': get_lifecycle(bucket_name, **conn),
-        'logging': get_logging(bucket_name, **conn),
-        'policy': get_policy(bucket_name, **conn),
         'region': region,
-        'tags': get_tags(bucket_name, **conn),
-        'versioning': get_versioning(bucket_name, **conn),
-        'website': get_website(bucket_name, **conn),
-        'cors': get_cors(bucket_name, **conn),
-        'notifications': get_notifications(bucket_name, **conn),
-        'acceleration': get_acceleration(bucket_name, **conn),
-        'replication': get_replication(bucket_name, **conn),
-        'analytics_configurations': get_bucket_analytics_configurations(bucket_name, **conn),
-        'metrics_configurations': get_bucket_metrics_configurations(bucket_name, **conn),
-        'inventory_configurations': get_bucket_inventory_configurations(bucket_name, **conn),
         '_version': 5
     }
-
-    if include_created:
-        result["Created"] = get_bucket_created(bucket_name, **conn)
+    
+    for entry in FlagRegistry.r:
+        if flags & entry['flag']:
+            result.update({entry['key']: entry['method'](bucket_name, **conn)})
+
+    if flags & FLAGS.GRANTS:
```

Comment:
See the comment for the `rtv_ix`.  It would be nice to use the same approach.  Maybe you can help figure out how to do it.